packages/chain-adapters/src/solana/constants.ts

```diff
@@ -0,0 +1,6 @@
+export const SOLANA_COMPUTE_UNITS_BUFFER_MULTIPLIER = 1.4
+
+export const SOLANA_MINIMUM_INSTRUCTIONS_NUMBER = 3
+
+// @TODO: this could change if they update it, it would be safer to use unchained using getRentExemptAmount
+export const SOLANA_MINIMUM_RENT_EXEMPTION_LAMPORTS = 890880
```

Comment:
[q]: What's the plan for this? Is the intention to leave this in or action it in follow-up?

Reply:
> Also, worth commenting where this comes from: https://github.com/coral-xyz/backpack/blob/5a538a41d060d2c48507007f96c766483115aecc/packages/common/src/constants.ts#L433

It's not coming from backpack, but updated the comment to make clear it comes from `getMinimumBalanceForRentExemption`

Reply:
> [q]: What's the plan for this? Is the intention to leave this in or action it in follow-up?

Honestly I think the effort is a bit big to add another unchained extra call for this, Solana told devs it could change but it never changed for now and everyone is basing their maths on this, I suspect they would at least take some months to update it

Even thought I would feel safer to use the RPC value so we will never have anything to change in the future 

Happy to discuss it in gang gang if needed, we could simply unblock this PR and action a follow up, happy with it too!

Reply:
> Also, worth commenting where this comes from: https://github.com/coral-xyz/backpack/blob/5a538a41d060d2c48507007f96c766483115aecc/packages/common/src/constants.ts#L433

Updated in [275f332](https://github.com/shapeshift/web/pull/8327/commits/275f332ecfa01fe8a67b10465016c05190d26415)